content/docs/2.10/scalers/prometheus.md

```diff
@@ -9,18 +9,18 @@ go_file = "prometheus_scaler"
 ### Trigger Specification
 
 This specification describes the `prometheus` trigger that scales based on a Prometheus.
-
+Description. (Values: x, y, z, Default: y, Optional, Extra Info)
```

Comment:
I think this was done by accident, right?
```suggestion

```

Reply:
Yup, I apologize.